### DISCUSSION TITLE
Can't get CURL statements formatted correctly to Post Shipment data (update tracking data & inventory).

### DISCUSSION THREAD

**Discussion by FulfillPlus** [Sellbrite/Sellbrite-API#36] [Q&A]
I’m having issues getting CURL statements configured to post shipping updates via API.  We have followed the layouts described in the API specs with no success.
 
We have a Batch file the executes the CURL and a separate Data File (see below)
**(BAB41860.BAT)**
CURL -k -X POST --user 7e18782b-43f5-4d6a-9b60-486c0f009577:ac8b81a689f9a51ce1cf47202d5cc660 HTTPS://API.SELLBRITE.COM/v1/shipments ^
-H "Content-Type: Application/json" ^
-d @BAB41860SHIPMENTPARAM.TXT ^
-o BAB-41860-ERROR.TXT
TIMEOUT /T 10

**(BAB41860shipmentparam.txt)**
{shipment: { sb_order_seq:41860, carrier_name:"USPS", tracking_number:"9400TESTORDER4186000000", items:[{ sku:"00-XF1Z-V9NH", quantity:,1}], shipping_method:"GROUND ADVANTAGE", warehouse_uuid:"47F66A49-D8FA-442A-911D-CBD9C7879D49"}}

**SELLBRITE API SPECS for Shipment POSTS show the following:**

POST https://api.sellbrite.com/v1/shipments

**Body Parament Specs**

{
  shipment: {
    sb_order_seq: int32, (REQUIRED, Sellbrite generated sequence number)
    carrier_name: String, (REQUIRED, Name of the carrier used to ship the order. ("UPS", "USPS", "FedEx", etc.))
    tracking_number: String, (REQUIRED, Tracking number for this shipment)
    items: [ (REQUIRED)
      {
        sku: String, (REQUIRED, SKU of what is being shipped (This needs to match the order item sku. If it does not match, we attempt to match on the inventory_sku. If neither match, an error will be raised))
        quantity: int32 (REQUIRED, Quantity that is being shipped)
      }
    ],
    shipping_method: String, (Shipping method of the shipment ("Priority", "2 Day", "Express" etc.))
    warehouse_uuid: String (Warehouse uuid of the warehouse the shipment is shipping from),
    tracking_url: String (Tracking url of the shipment)
  }
}
 
Can anyone tell me what is wrong with our CURL or Data files?

**Comment by rflores1-godaddy** [comment]
Hello @FulfillPlus,

Thank you for reaching out to us. We're not sure what specific issue you're facing, could you share the full response that you're getting from our API so that we can provide you further assistance?

Also, I've noticed that the syntax for the items in your .txt file may have been mistyped: `items:[{ sku:"00-XF1Z-V9NH", quantity:,1}]` should be `items:[{ sku:"00-XF1Z-V9NH", quantity:1}]`. This could be part of the issue you are facing.

**Comment by FulfillPlus** [comment]
I figured it all out by trial and error.

I’ve attached then new files.

I’d recommend showing examples of data layouts for orders with multiple line items.

CG
